src/cmdline.rs

```diff
@@ -397,9 +397,24 @@ pub struct ListStreamConfig {
     pub stream_base_url: Option<Url>,
 }
 
+#[derive(Debug, StructOpt)]
+pub struct CommonCustomizeConfig {
+    /// Script to run before installation
+    #[structopt(long, value_name = "path")]
```

Comment:
xref https://github.com/coreos/coreos-installer/pull/305#discussion_r460055902 :grin: 

Consistency seems good, yes.  I still find SCREAMING CASE less readable, and it seems unnecessary since clap wraps value names in `<>` anyway.  But if you feel strongly about it, I can convert these over.